src/data/set/prod.lean

```diff
@@ -40,10 +40,18 @@ lemma mem_prod_eq {p : α × β} : p ∈ s ×ˢ t = (p.1 ∈ s ∧ p.2 ∈ t) :=
 
 @[simp] lemma mem_prod {p : α × β} : p ∈ s ×ˢ t ↔ p.1 ∈ s ∧ p.2 ∈ t := iff.rfl
 
+@[simp] lemma not_mem_prod {p : α × β} : p ∉ s ×ˢ t ↔ p.1 ∉ s ∨ p.2 ∉ t := not_and_distrib
+
```

Comment:
The linter will reject `simp` on this lemma, since it's proved via `mem_prod` (`∉` is just notation). I don't think this lemma is worth having anyway, but another reviewer might disagree

Reply:
I'd only added it to simplify the more complicated proof of the decidable instance, with that simpler proof I don't think it's needed at all